content-module0.md

```diff
@@ -0,0 +1,141 @@
+# Module 0
+
+# Development environment
+
+The tools you will need to effectively develop the API
+
+## Node
+
+Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. This means we write our code in JavaScript/TypeScript.
+
+Node.js should **never** be installed independently on your machine.  
+Instead you should install it through [nvm (node version manager)](https://formulae.brew.sh/formula/nvm) or [fnm](https://github.com/Schniz/fnm).  
+
+### NVM
+
+Install nvm with [nvm](https://formulae.brew.sh/formula/nvm) [using homebrew](https://brew.sh/) (or [on windows](https://github.com/coreybutler/nvm-windows/releases)).
+
+For nvm you may need to edit your shell configuration file (`.bashrc`, `.zshrc`, etc.) to load nvm.
+
+To know your current shell run:
+
+```bash
+echo $SHELL
+```
+
+Usually, it will be `zsh`;
+
+To edit your shell configuration file run:
+
+```bash
+vim ~/.zshrc
+```
+
+Press `i` to enter the `insert` mode and paste the following lines to the end of the file:
+
+```bash
+export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
+```
+
+Then press `esc` and type `:wq` to save and exit.
+
+_If Node.js is already installed it should be removed before installing nvm._
+
+When the installation is completed you can simply run the following to install latest LTS node (currently 18.14.0):
+
+```bash
+nvm install --lts
+```
+
+### FNM
+
+Install fnm by executing
+```shell
+curl -fsSL https://fnm.vercel.app/install | bash
+```
+
+add completions
+```shell
+fnm completions --shell <SHELL>
+```
```

Comment:
I would only give them one option, Aside from @doichev-kostia  I think nobody ever used fnm so I would advise against it for now.